message.go

```diff
@@ -119,6 +127,7 @@ func (c *Client) MessageMark(ctx context.Context, r MessageMarkRequest) (*Messag
 	return &MessageMarkReply{}, nil
 }
 
+// MessageDelete performs a Delete request.
 func (c *Client) MessageDelete(ctx context.Context, r MessageDeleteRequest) (*MessageDeleteReply, error) {
```

Comment:
A comment like, `MessageDelete will permanently remove a message from the SPVServer` could be a bit more descriptive.

Also, what happens if the message isn't found or is already deleted, is an error returned, if so what type?